### ISSUE TITLE
[Huawei Mediapad M5 Lite] Bootloop, logcat full of Fatals and Errors

### ISSUE THREAD

**Issue by Hacker432** [phhusson/treble_experimentations#1234]
Can't boot the device, it stays on Android boot logo.
I flashed system-quack-arm64-aonly-vanilla because gapps included version was too large for my device
See: https://bit.ly/2wXMeIZ (it was version 211)

Logcat attached:
[logcat.txt](https://github.com/phhusson/treble_experimentations/files/4437810/logcat.txt)

**Comment by fourofspades** [comment]
Did you try using the treblechecker app, as my Mediapad M5 uses arm64 AB

**Comment by Hacker432** [comment]
![Screenshot_20200407-102734](https://user-images.githubusercontent.com/16880178/78660597-a760f700-78cd-11ea-9bc5-92f1cff2bc2e.jpg)
I can confirm it's A-only
Maybe I should revert to stock firmware, I mean the old one not the latest

**Comment by fourofspades** [comment]
Weird, my Mediapad M5 is different. Perhaps there are large region differences.

![Screenshot_20200407-115114](https://user-images.githubusercontent.com/61576868/78661329-847f1480-78c6-11ea-80de-1e7171ce8c58.png)

**Comment by phhusson** [comment]
That's because of vendor version.
They have EMUI8, which is A-only, and ou have EMUI9 which is SaR/Ab

Le mar. 7 avr. 2020 à 12:54, fourofspades <notifications@github.com> a
écrit :

**Comment by Hacker432** [comment]
![Screenshot_20200407-190911](https://user-images.githubusercontent.com/16880178/78698932-5f5cc700-7903-11ea-805b-08ac8151df68.png)
Up and running.
As I suspected I had to revert to an older firmware.
Here was suggested B177 ([Android Hilfe](https://www.android-hilfe.de/forum/huawei-mediapad-m5-lite.3470/custom-roms-mediapad-m5-lite.909217-page-2.html)).
So I reverted to B177 using HuRUpdater and Huawei Firmware Finder and then followed the guide for my device on xda. I used the vanilla version, but I noticed using Diskinfo that my system partition is now 3.5GB (was 2.5GB) don't know if it's firware related...so maybe I could have installed the gapps included version too
I had a problem with the internal storage at first boot, the partition was recognized as unknown and the apps couldn't write.
Had to boot in twrp and wipe internal storage.
